### PR TITLE
Create readme file

### PR DESCRIPTION
**Pull request by dominikhajduk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Task Manager
+Self learning project
+
+
+## Installation
+Clone repository
+
+    docker-compose up
```
Comment (partikus) [inline README.md]: missing code block

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md)
```
Comment (partikus) [inline README.md]: please add license as well

**Review by partikus — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dominikhajduk) [reply README.md]: 4 spaces will be changed to code block. Or should we force ` ``` ` ?

File: README.md (hunk 2 above)
Comment (dominikhajduk) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (mateuszmaslanka) [reply README.md]: IMO` ```bash`

**Comment by dominikhajduk** [comment]
@mateuszmaslanka @partikus improved

**Review by tarnawski — APPROVED** [review]
Looks nice. Thanks for update!

**Comment by tarnawski** [comment]
@partikus pls check again

File: README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Task Manager
+Self learning project
+
+
+## Installation
+Clone repository
+
+```bash
+    docker-compose up
```
Comment (partikus) [inline README.md:9]: useless indentation

**Review by partikus — APPROVED** [review]

**Pull request merged**